Makefile

```diff
@@ -236,8 +236,13 @@ dist/argoexec.image: $(ARGOEXEC_PKGS)
 # generation
 
 .PHONY: codegen
-codegen: \
-	pkg/apis/workflow/v1alpha1/generated.proto \
+codegen: codegen-types codegen-swagger codegen-docs
```

Comment:
Individual targets

Reply:
could we have `make types` `make swagger` and `make docs`?

Reply:
Yup!